sqlx_user_crud/tests/controller_test/group_controller_test.rs

```diff
@@ -0,0 +1,87 @@
+use super::{init_app_state, randomize_string};
+use sqlx_user_crud::controller;
+use actix_web::{http, test, App};
+use sqlx_user_crud::controller::group_controller::GroupUpdate;
+use sqlx;
+
+#[actix_rt::test]
```

Comment:
:+1: for all these tests!

Reply:
It looks like the tests in the pipeline are a problem. They rely on a MySQL db with a specific database configuration being up and running. Is there a way to make this work with the pipeline? What are our options?

Reply:
You can let GitHub actions [set up an auxilliary docker container for your job](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices).

Reply:
Launching a database in a separate docker container is far outside the scope of what should be included in actix/examples tests.  This exact scenario is what compels programmers to avoid using sqlx in the first place.

You are free to include whatever tests you like from your own project repo.  Your repo can be indexed from the actix/examples README.md.  Or, omit the database tests and leave the rest that pertain to the web server.

Reply:
I don't see what this has to do with the compile-time verification supported by SQLx, it is not being used here at all.

Reply:
I'm conveying a general guideline for PRs submitted to this repo.  An example requiring an environment involving more than an actix-web server can be documented in the repo, as illustrated in some of the other db related examples, or setup however the author likes from an independently managed repo that we can index to.

Reply:
I agree with @Dowwie on this. Since this repository is Actix Examples, Actix is the only true dependency. Even if I implemented the docker container for the database, that would impose the requirement of docker on anyone who cloned the repo and wanted to run the full test suite. If this repository was Actix and Sqlx examples, this would be a different situation. I wasn't considering the pipeline requirements when I originally made the PR.

I'll abandon this and add a link to this project to the index.